### PR TITLE
Add Georgian (ka) locale

### PR DESCRIPTION
**Pull request by akalongman**
This PR adds Georgian (ka) translations

### CODE REVIEW

**Comment by akalongman** [comment]
Failing tests are not related to my changes, looks like they are broken. Same issue https://github.com/fullcalendar/fullcalendar/pull/4023#issuecomment-359951127

**Comment by acerix** [comment]
Thanks for the PR

File: locale/ka.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import 'moment/locale/ka';
+import * as FullCalendar from 'fullcalendar';
+
+
+/* Georgian initialisation for the jQuery UI date picker plugin. */
+/* Written by Avtandil Kikabidze aka LONGMAN <akalongman@gmail.com>. */
+FullCalendar.datepickerLocale('ka', 'ka', {
```
Comment (arshaw) [inline locale/ka.js]: are the translations in momentjs not satisfactory?

https://github.com/moment/moment/blob/develop/locale/ka.js#L13

if they are alright, and you don't need to override them, you can omit them from this file. will fall back to moment

Comment (arshaw) [reply locale/ka.js]: nevermind. stupid mistake of mine. this is good

**Comment by arshaw** [comment]
released in 3.8.1, thanks

**Comment by akalongman** [comment]
Thank you :+1:

**Pull request merged**